src/actors/build/cx_actor.jl

```diff
@@ -886,7 +886,11 @@ function optimize_layer_outline(
         func = shape_function(shape; resolution)
         initial_clerance = max(hfs_thickness, lfs_thickness) * vertical_clearance
         shape_parameters0 = initialize_shape_parameters(shape, oR, oZ, l_start, l_end, initial_clerance)
-        shape_parameters = optimize_outline(oR, oZ, hfs_thickness, lfs_thickness, func, l_start, l_end, shape_parameters0)
+        if occursin("TF",layer.name)
+            shape_parameters = shape_parameters0
+        else
+            shape_parameters = optimize_outline(oR, oZ, hfs_thickness, lfs_thickness, func, l_start, l_end, shape_parameters0)
+        end
```

Comment:
I also don't agree with this change. How can you know the right parameters for the TF layer if you don't optimize them?

Reply:
I added this change because the princeton_D_scaled TF shape was being incorrectly changed by this line. I'm my experience, it makes more sense to have the TF shape be dictated by the user instead of optimized... But to be fair I've only been working with one specific case, and I don't know if this is generalizable.

Reply:
The optimizer will try to make the layers (including the TF) be as snug as possible. Do you have an example of it failing with `princeton_D_scaled`? what kind of failure were you seing?

Reply:
Before my fix:
![Screen Shot 2024-11-20 at 11 00 05 AM](https://github.com/user-attachments/assets/d27b3120-456f-4c7e-814c-5a9f568825ca)

After my fix:
![Screen Shot 2024-11-20 at 11 02 18 AM](https://github.com/user-attachments/assets/c2a27db8-574b-449b-9f61-27d78739ee87)